src/components/solvers/TextInputMask.tsx

```diff
@@ -3,13 +3,10 @@ import Head from "next/head";
 import React, { ReactElement, useState } from "react";
 import { Container } from "../Container";
 import { Main } from "../Main";
-import { InputButtonPanel } from "./buttons/InputButtonPanel";
 import { Help } from "./SAT/Help";
-import { SolverTitle } from "./SolverTitle";
+import { EditorControls } from "./EditorControls";
 
 export interface TextInputMaskProperties {
```

Comment:
We should maybe add documentation for all props in this interface. Not necessary to merge this PR, just in general

Reply:
now tracked in https://github.com/ProvideQ/toolbox-web/issues/32